apps/src/sites/studio/pages/levels-gamelab-main.js

```diff
@@ -1,8 +1,8 @@
-var appMain = require('../appMain');
-var studioApp = require('../StudioApp').singleton;
-var GameLab = require('./GameLab');
-var skins = require('./skins');
-var levels = require('./levels');
+import appMain from "@cdo/apps/appMain";
+import {singleton as studioApp} from '@cdo/apps/StudioApp';
```

Comment:
Pretty sure this is okay, but I can never remember which interop case doesn't work; StudioApp.js is exporting stuff this way: `module.exports = { singleton: new StudioApp() };`.  That's fine to ES6-import like this?